srcCxx/shape_configurator_eprosima_fast_dds.h

```diff
@@ -32,6 +32,9 @@
 namespace DDS = eprosima::fastdds::dds;
 #define RETCODE_OK DDS::RETCODE_OK
 
+#define DDS_BOOLEAN_TRUE                   true
```

Comment:
We usually put default definitions [here](https://github.com/omg-dds/dds-rtps/blob/9e1fdbcd30bc785e5e5113a94e5e936695a3385e/srcCxx/shape_main.cxx#L36-L59)